src/libs/runtime-utils.js

```diff
@@ -21,13 +21,32 @@ export const computeStyles = {
   warningView: { backgroundColor: colors.warning(0.1) }
 }
 
+export const pdTypes = {
+  standard: {
+    label: 'pd-standard',
+    displayName: 'Standard',
+    regionToPricesName: 'monthlyStandardDiskPrice'
+  },
+  ssd: {
+    label: 'pd-ssd',
+    displayName: 'Solid state drive (SSD)',
+    regionToPricesName: 'monthlySSDDiskPrice'
+  },
+  fromString: str => Utils.switchCase(str,
+    [pdTypes.standard.label, () => pdTypes.standard],
+    [pdTypes.ssd.label, () => pdTypes.ssd],
+    [Utils.DEFAULT, () => console.error('Should not be calling pdTypes. fromString for $(str), invalid disk type')]
```

Comment:
```suggestion
    [Utils.DEFAULT, () => console.error(`Invalid disk type: Should not be calling pdTypes.fromString for ${str}`)]
```

to use back-tick since we substitute the value of `str`. May also be more readable with the `Invalid disk type` piece at the beginning?